node/src/actors/json_rpc/api.rs

```diff
@@ -145,6 +145,12 @@ pub fn attach_regular_methods<H>(
     server.add_actix_method(system, "queryStakes", |params: Params| {
         Box::pin(query_stakes(params.parse()))
     });
+    server.add_actix_method(system, "ranks", |params: Params| {
```

Comment:
Just nitpicking here. What you get in return of this method is a list comprising all stake entries, ranked by power. Let me argue that's a "ranking" and not a "ranks" inasmuch it's not a list of ranks (like in military ranks, etc) but more of a ranked / sorted kind of roster.

```suggestion
    server.add_actix_method(system, "ranking", |params: Params| {
```

Alternatively, if we follow the pattern of naming the method with the action it will perform instead of what we are expecting it to return (like in methods named `getX` or `queryX`), then `rank` could make sense too:

```suggestion
    server.add_actix_method(system, "rank", |params: Params| {
```

Reply:
Agreed, perhaps a better name would be "queryPowers":
- as queryStakes was not nitpicked
- and because this new method doesn't actually return a list of stake entries, but tuples containing the validator, withdrawer and current power for specified capability